pkg/smart/parse.go

```diff
@@ -245,36 +257,40 @@ func smartCtlParse(raw []string) (common.MeasurementsMap, []error) {
 	return marshaledDisks, errs
 }
 
-func smartctlIsSupportedVersion(buildStr string) error {
+func smartctlIsSupportedVersion(buildStr string) (string, error) {
 	if !smartctlVersionRegexp.Match([]byte(buildStr)) {
-		return errors.New("smart: couldn't detect smartctl version")
+		return "", errors.New("smart: couldn't detect smartctl version")
 	}
 
 	ver := smartctlVersionRegexp.FindAllStringSubmatch(buildStr, -1)
 
 	if len(ver) < 1 && len(ver[0]) < 2 {
-		return ErrParseSmartctlVersion
+		return "", ErrParseSmartctlVersion
 	}
 
 	tok := strings.Split(ver[0][1], ".")
 
 	if len(tok) != 2 {
-		return ErrParseSmartctlVersion
+		return "", ErrParseSmartctlVersion
 	}
 
 	var major int
+	var minor int
 	var err error
 
-	major, err = strconv.Atoi(tok[0])
-	if err != nil {
-		return fmt.Errorf("smart: parse smartctl version: %s", err)
+	if major, err = strconv.Atoi(tok[0]); err != nil {
+		return "", fmt.Errorf("smart: parse smartctl version: %s", err)
+	}
+
+	if minor, err = strconv.Atoi(tok[1]); err != nil {
+		return "", fmt.Errorf("smart: parse smartctl version: %s", err)
 	}
 
 	if major >= 7 {
```

Comment:
Probably should be moved to `const` value.

Reply:
done